src/test/java/wolox/training/controllers/UserControllerTest.java

```diff
@@ -0,0 +1,74 @@
+package wolox.training.controllers;
+
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.WebMvcTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.http.MediaType;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.request.MockMvcRequestBuilders;
+import wolox.training.repositories.IUserRepository;
+
+@WebMvcTest(UserController.class)
+public class UserControllerTest {
+
+    @Autowired
+    private MockMvc mvc;
+
+    @MockBean
+    private UserController userController;
+
+    @MockBean
+    private IUserRepository userRepository;
+
+    @Test
+    public void givenUsers_whenGetUsers_thenReturnAllUsers() throws Exception {
+        mvc.perform(MockMvcRequestBuilders.get("/users").contentType(MediaType.APPLICATION_JSON_VALUE)
+                        .content(""))
+                .andExpect(status().isOk());
```

Comment:
The mocks for `userRepository` in the tests are missing. For example:
```suggestion
        doReturn(users).when(userRepository).findAll();
        mvc.perform(MockMvcRequestBuilders.get("/users").contentType(MediaType.APPLICATION_JSON_VALUE))
                .andExpect(status().isOk());
```